packages/hebao_v1/contracts/modules/security/RecoveryModule.sol

```diff
@@ -30,7 +30,6 @@ import "./SecurityModule.sol";
 /// @title RecoveryModule
 contract RecoveryModule is SecurityModule
 {
-
```

Comment:
In extractMetaTxSigners function.

require(data.length >= (100 + 32 * numSigners), "DATA_INVALID");

looks like should be: "require(data.length >= (132 + 32 * numSigners), "DATA_INVALID");"

Reply:
When using `.length` you don't count `data_length` as part of the array length, that's just how arrays store their length internally (so when you just use solidity you don't need to know this). It is important to keep `data_length` in mind when using assembly however.